parser.py

```diff
@@ -11,11 +11,13 @@
         commits[info["data"]["commit"]] = info
         parents[info["data"]["commit"]] = [commits[parent] for parent in info["data"]["parents"]]
 
-root = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]
+a = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]
```

Comment:
Why did you choose this name? This could be more sensible.

Reply:
I will change this to `parent = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]`